OutOfSchool/OutOfSchool.Tests/ParentServiceTest.cs

```diff
@@ -0,0 +1,196 @@
+﻿using AutoMapper;
+using Microsoft.EntityFrameworkCore;
+using NUnit.Framework;
+using OutOfSchool.Services;
+using OutOfSchool.Services.Models;
+using OutOfSchool.Services.Repository;
+using OutOfSchool.WebApi.Models;
+using OutOfSchool.WebApi.Services.Implementation;
+using OutOfSchool.WebApi.Services.Interfaces;
+using OutOfSchool.WebApi.Services.Mapping;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OutOfSchool.Tests
+{
+    [TestFixture]
+    public class ParentServiceTest
+    {
+        private OutOfSchoolDbContext _context;
+        private IEntityRepository<Parent> _entityRepository;
+        private IParentService _service;
+        private IMapper _mapper;
+
+        [SetUp]
+        public void SetUp()
+        {
+            var options = new DbContextOptionsBuilder<OutOfSchoolDbContext>().UseInMemoryDatabase(databaseName: "Test").Options;
+            _context = new OutOfSchoolDbContext(options);
+            _entityRepository = new EntityRepository<Parent>(_context);
+            var parentMapper = new MapperConfiguration(x => x.AddProfile(new ParentMapperProfile())).CreateMapper();
+            _mapper = parentMapper;
+            _service = new ParentService(_entityRepository, parentMapper);
+        }
+
+        [Test]
+        public void TestCreate()
+        {
+            var parent1 = new ParentDTO { FirstName = "Test", MiddleName = "Test", LastName = "Test" };
```

Comment:
please follow test structure
you can refer to Bohdan's approach:
https://github.com/ita-social-projects/OoS-Backend/pull/42/files#diff-a0eb263dbc705f901abb9964479e439c9ddcc887d470a6eef345bf078ab2ed57

test structure:
// Arrange

            // Act

            // Assert